.github/workflows/android-ci.yml

```
@@ -0,0 +1,47 @@
+                with:
+                    gradle-home-cache-cleanup: true
+
+            -   name: Generate local.properties
+                run: |
+                    echo '${{ secrets.LOCAL_PROPERTIES }}' >> ./local.properties
+
+            -   name: Code style checks
+                run: |
+                    ./gradlew ktlintCheck detekt
+
+            -   name: Run build
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)